contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -57,7 +57,10 @@ def prepare(cls, options, round_manager):
 
   @classmethod
   def register_options(cls, register):
-    register('--mypy-version', default='0.720', help='The version of mypy to use.')
+    register('--mypy-version', default='0.720', help='The version of mypy to use.',
+             removal_version=cls.deprecated_options_scope_removal_version,
```

Comment:
We'll want to wait 2 full versions from now. `cls.deprecated_options_scope_removal_version` is referring to how we've deprecated `./pants mypy` and you instead should use `./pants lint.mypy`. 

The most current version is 1.22.0.dev0 per https://github.com/pantsbuild/pants/blob/a921f4dde2d7f62341a63dc6eed50cf29e32c0d8/src/python/pants/VERSION#L1

So, this should be removed in 1.24.0.dev1.

Reply:
Ah, that makes sense. Thanks!